WordPress/Classes/ViewRelated/Blog/My Site/NoSitesView.swift

```diff
@@ -0,0 +1,188 @@
+import SwiftUI
+
+protocol NoSitesViewDelegate: AnyObject {
+    func didTapAccountAndSettingsButton()
+}
+
+struct AddNewSiteConfiguration {
+    let canCreateWPComSite: Bool
+    let canAddSelfHostedSite: Bool
+    let launchSiteCreation: () -> Void
+    let launchLoginForSelfHostedSite: () -> Void
+}
+
+struct NoSitesView: View {
+
+    @SwiftUI.Environment(\.colorScheme) var colorScheme
+
+    @State private var isShowingDialog = false
+
+    let viewModel: NoSitesViewModel
+    let addNewSiteConfiguration: AddNewSiteConfiguration
+
+    weak var delegate: NoSitesViewDelegate?
+
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            Color(uiColor: .listBackground)
+                .edgesIgnoringSafeArea(.all)
+
+            makeMainView()
+                .padding(.horizontal, 32)
+
+            if viewModel.isShowingAccountAndSettings {
+                makeAccountAndSettingsButton()
+                    .padding(.horizontal, 32)
+                    .padding(.bottom, 20)
```

Comment:
Is 20 the correct spacing on designs? I think we should be using 8 multipliers.

Reply:
On Figma it looks like its 22 - but I can bump it to 24? cc: @osullivanchris

Reply:
Bumped to 24 [d52a04e](https://github.com/wordpress-mobile/WordPress-iOS/pull/21456/commits/d52a04e5272a5e7756318a28cd4ae76c1aeef922)